src/OrbitGgp/ClientTest.cpp

```diff
@@ -35,6 +35,17 @@ class OrbitGgpClientTest : public testing::Test {
   OrbitGgpClientTest() : main_thread_executor_(orbit_qt_utils::MainThreadExecutorImpl::Create()){};
 
  protected:
+  std::unique_ptr<Client> CreateWorkingClient() {
+    auto create_result = CreateClient(QString::fromStdString(mock_ggp_working_.string()));
+    EXPECT_THAT(create_result, HasValue());
```

Comment:
ASSERT_THAT?

Reply:
ASSERT is here not possible, since the macro contains a `return;` and thats incompatible with return type std::unique_ptr<Client>. I am using a CHECK instead. yaqs/q/5581738191355904

Reply:
Using CHECK is not really a solution because it will kill the process and that means no output XML file will be generated. So this is the same effect as calling `value()` on an ErrorMessageOr instance. (I made the comment mainly to ensure that the test executable won't crash)

Reply:
Not sure what the best option is in that case.

Reply:
I removed the whole commit